.github/workflows/create-reports.yaml

```diff
@@ -30,11 +30,19 @@ jobs:
       uses: Open-Systems-Pharmacology/Workflows/.github/actions/get-tools-versions@main
       with:
         tools-path: tools.csv
-        
+
+    - id: install-r
+      name: Setup R
+      uses: r-lib/actions/setup-r@v2
+      with:
+        r-version: 'latest'

```

Comment:
why do we have here the latest version and not the one returned by get-tools-versions?